service/src/main/java/bio/terra/workspace/common/utils/RetryRules.java

```diff
@@ -0,0 +1,51 @@
+package bio.terra.workspace.common.utils;
+
+import bio.terra.stairway.RetryRule;
+import bio.terra.stairway.RetryRuleExponentialBackoff;
+import bio.terra.stairway.RetryRuleFixedInterval;
+
+public class RetryRules {
+
+  private RetryRules() {}
+
+  /**
+   * Retry rule for immediately retrying a failed step twice. This is useful for retrying operations
+   * where waiting is not necessary, e.g. retrying conflicted SQL transactions.
+   */
+  public static RetryRule databaseConflict() {
```

Comment:
When @zloery finishes his database retry work, we won't want this one either.
I'd suggest removing it and its usage now.

Reply:
👍🏻